### ISSUE TITLE
Adding insecure registry doesn't work 

### ISSUE THREAD

**Issue by dharmit** [openshift/openshift-ansible#8665]
#### Description

I'm adding following to my hosts file to ensure that insecure registry is added to my cluster:
```
openshift_docker_insecure_registries=172.29.33.8:5000
openshift_docker_additional_registries=172.29.33.8:5000
```

##### Version

* Your ansible version per `ansible --version`

* The output of `rpm -q openshift-ansible`

```
$ ansible --version
ansible 2.5.3
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /bin/ansible
  python version = 2.7.5 (default, Apr 11 2018, 07:36:10) [GCC 4.8.5 20150623 (Red Hat 4.8.5-28)]

$ rpm -qa| grep openshift-ansible
openshift-ansible-docs-3.9.0-0.53.0.git.1.af49d87.el7.noarch
openshift-ansible-roles-3.9.0-0.53.0.git.1.af49d87.el7.noarch
openshift-ansible-playbooks-3.9.0-0.53.0.git.1.af49d87.el7.noarch
openshift-ansible-3.9.0-0.53.0.git.1.af49d87.el7.noarch
```

##### Steps To Reproduce
1. Add `openshift_docker_insecure_registries` and `openshift_docker_additional_registries` parameters to hosts file
2. Execute the playbook to spin up a cluster

##### Expected Results

Insecure registries added to the cluster

##### Observed Results
`docker push` to configured insecure registries fails. Don't see insecure registry configuration set in either `/etc/docker/daemon.json` or `/etc/sysconfig/docker`. The command used to start docker daemon doesn't have any parameter for insecure registry.

##### Additional Information

* Your operating system and version, ie: RHEL 7.2, Fedora 23 (`$ cat /etc/redhat-release`)
* Your inventory file (especially any non-standard configuration parameters)
* Sample code, etc

```
$ cat /etc/os-release
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"
```

[Link to hosts file](https://gist.github.com/dharmit/61fc85394fa3932626f6a7f196e9d946)

Am I doing something wrong here?

**Comment by vrutkovs** [comment]
Which playbooks are being run? IIRC `prerequisites` would restart docker and the changes to `INSECURE_REGISTRY` would be applied

**Comment by dharmit** [comment]
I'm running only this playbook - `/usr/share/ansible/openshift-ansible/playbooks/deploy_cluster.yml`

**Comment by vrutkovs** [comment]
Does it work if both `/usr/share/ansible/openshift-ansible/playbooks/prerequisites.yml` and `/usr/share/ansible/openshift-ansible/playbooks/deploy_cluster.yml` are run?

**Comment by dharmit** [comment]
Yes, running the `prerequisites.yml` before `deploy_cluster.yml` does the trick. Thanks for your help!
